api/applications/applicationRouter.js

```diff
@@ -6,15 +6,29 @@ const router = express.Router();
 const jwt = require('jwt-decode');
 const { adminRequired } = require('../middleware/permissionsRequired.js');
 
-router.get('/', authRequired, adminRequired, (req, res, next) => {
-  Application.getPendingTickets()
+// get all pending mentee applications
+
+router.get('/mentee', authRequired, adminRequired, (req, res, next) => {
+  Application.getPendingMenteeTickets()
     .then((applicationList) => {
       res.status(200).json(applicationList);
     })
     .catch(next);
 });
 
-router.post('/', authRequired, function (req, res, next) {
+// get all pending mentor application
+
+router.get('/mentor', authRequired, adminRequired, (req, res, next) => {
+  Application.getPendingMentorTickets()
+    .then((applicationList) => {
+      res.status(200).json(applicationList);
+    })
+    .catch(next);
+});
+
+// post a new application for the current logged in user
+
+router.post('/new-application', authRequired, function (req, res, next) {
```

Comment:
Does a newly applying user _need_ to be logged in to submit an app? 🤔

Reply:
Yes, a new user would need to be signed in with their OKTA login first before they can apply for a role. By default a new user has the "pending" role until they apply for their desired role, whether that is "mentor" or "mentee"